client/ayon_houdini/plugins/create/create_FBX.py

```diff
@@ -1,32 +1,34 @@
 # -*- coding: utf-8 -*-
-"""Creator for Unreal Static Meshes."""
+"""Creator plugin for creating Model product type as FBX."""
 from ayon_houdini.api import plugin
 from ayon_core.lib import BoolDef, EnumDef
 
 import hou
 
 
-class CreateStaticMesh(plugin.HoudiniCreator):
-    """Static Meshes as FBX. """
-
-    identifier = "io.openpype.creators.houdini.staticmesh.fbx"
-    label = "Static Mesh (FBX)"
-    product_type = "staticMesh"
-    icon = "fa5s.cubes"
+class CreateFBX(plugin.HoudiniCreator):
+    """Model as FBX. """
 
+    identifier = "io.openpype.creators.houdini.modelFBX.fbx"
+    label = "model (FBX)A"
+    product_type = "model"
+    icon = "cube"
     default_variants = ["Main"]
 
     # Default render target
     render_target = "local"
 
+    def get_publish_families(self):
+        return ["fbx", "model"]
+
     def create(self, product_name, instance_data, pre_create_data):
 
         instance_data.update({"node_type": "filmboxfbx"})
         creator_attributes = instance_data.setdefault(
             "creator_attributes", dict())
         creator_attributes["render_target"] = pre_create_data["render_target"]
 
-        instance = super(CreateStaticMesh, self).create(
+        instance = super(CreateFBX, self).create(
```

Comment:
This should be fine in Py3.
```suggestion
        instance = super().create(
```